examples/feature_showcase/checks.rs

```diff
@@ -103,6 +103,29 @@ pub async fn cooldowns(ctx: Context<'_>) -> Result<(), Error> {
     Ok(())
 }
 
+async fn random_cooldowns(ctx: Context<'_>) -> Result<bool, Error> {
+    ctx.command()
+        .cooldowns
+        .lock()
+        .unwrap()
+        .set_config(CooldownConfig {
+            global: Some(Duration::from_secs_f32(rand::random()) * 10),
+            user: Some(Duration::from_secs_f32(rand::random()) * 10),
+            guild: Some(Duration::from_secs_f32(rand::random()) * 10),
+            channel: Some(Duration::from_secs_f32(rand::random()) * 10),
+            member: Some(Duration::from_secs_f32(rand::random()) * 10),
+            __non_exhaustive: (),
```

Comment:
Kinda awkward to force users to include this. Should we perhaps make this struct not `non_exhaustive`? Or create a `new` function?

Reply:
Good point. Should probably add `Command.config(&self) -> CooldownConfig` too so you can modify the config incrementally instead of having to build one from scratch

I just checked and CooldownConfig has a Default impl too, so that's also a possibility to avoid the __non_exhaustive